src/patient-search/patient-search.component.tsx

```diff
@@ -1,10 +1,96 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import { match } from "react-router";
+import { debounce } from "lodash";
+import styles from "./patient-search.component.css";
+import { performPatientSearch } from "./patient-search.resource";
+import PatientSearchResults from "./patient-search-result/patient-search-result.component";
 
 export default function PatientSearch(props: PatientSearchProps) {
-  return <section>The patient search!</section>;
+  const searchTimeout = 400;
+  const [searchResults, setSearchResults] = useState([]);
+  const [emptyResult, setEmptyResult] = useState(false);
+  const [searchTerm, setSearchTerm] = useState("");
+
+  useEffect(() => {
+    if (searchTerm) {
+      performPatientSearch(searchTerm, "full").then(response => {
+        const { data } = response;
+        setSearchResults(data["results"]);
+        if (data["results"].length === 0) {
+          setEmptyResult(true);
+        } else {
+          setEmptyResult(false);
+        }
+      });
+    } else {
+      setSearchResults([]);
+    }
+  }, [searchTerm]);
+
+  const handleChange = debounce(searchTerm => {
+    setSearchTerm(searchTerm);
+  }, searchTimeout);
+
+  return (
+    <div>
```

Comment:
This is what in React is called a "useless div". Instead of useless divs, you can use [React fragments](https://reactjs.org/docs/react-api.html#reactfragment)